packages/x-charts-pro/src/ChartsToolbarPro/internals/ChartsToolbarDivider.tsx

```diff
@@ -2,13 +2,14 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import { styled } from '@mui/system';
 import { NotRendered, useChartsSlots } from '@mui/x-charts/internals';
-import { ChartsBaseSlotsPro } from '../../internals/slots/chartsBaseSlots';
-import { ChartBaseDividerProps } from '../../internals/slots/chartBaseSlotProps';
+import { ChartsBaseSlotPropsPro } from '../../internals/slots/chartsBaseSlots';
 
-export interface ChartsToolbarDividerProps extends ChartBaseDividerProps {}
+export type ChartsToolbarDividerProps = ChartsBaseSlotPropsPro['baseDivider'];
```

Comment:
Imo it is better to extend the type + the overrides than the prop of the slots since things get more explicit.